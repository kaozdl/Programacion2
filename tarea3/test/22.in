# remover_binario
remover_binario q .
insertar_binario (15,q) .
remover_binario q .
insertar_binario (14,c) .
remover_binario q .
insertar_binario (14,c) (16,d) (13,t) (5,z) .
remover_binario d c .
insertar_binario (1,u) .
remover_binario c u t .
insertar_binario (16,p) (13,j) (5,m) (6,a) (1,d) (3,c) .
remover_binario j .
Fin
