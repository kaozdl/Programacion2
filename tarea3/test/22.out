1>
2>
3>
(15,q)
0
0
4>
5>
(14,c)
0
0
6>
(14,c)
7>
---(5,z)
--(13,t)
-(16,d)
(14,c)
0
3
8>
(5,z)
(13,t)
9>
-(1,u)
(13,t)
0
1
10>
11>
(16,p)
--(5,m)
-(13,j)
---(1,d)
----(3,c)
--(6,a)
4
0
12>
(16,p)
-(5,m)
(1,d)
--(3,c)
-(6,a)
13>Fin.
